.changeset/stupid-drinks-burn.md

```diff
@@ -0,0 +1,8 @@
+---
+"@layerzerolabs/ua-devtools-evm-hardhat": patch
+"@layerzerolabs/devtools-evm-hardhat": patch
+"@layerzerolabs/io-devtools": patch
+"@layerzerolabs/devtools": patch
+---
```

Comment:
I think we also need to include `@layerzerolabs/ua-devtools-evm-hardhat-test`

Reply:
That's a private package we don't publish - we can but don't need to add changesets since well they have no effect